# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../neofetch/t2.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

From 83a92ca5c20e262c254c0d6084a29d408024c2ae Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ren=C3=A9=20Rebe?= <rene@exactcode.com>
Date: Fri, 7 Feb 2020 19:36:21 +0100
Subject: [PATCH] initial t2 support

---
 neofetch | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/neofetch b/neofetch
index 6bf0a596..cb69f57b 100755
--- a/neofetch
+++ b/neofetch
@@ -997,6 +997,12 @@ get_distro() {
                     *) distro="GoboLinux $(< /etc/GoboLinuxVersion)"
                 esac
 
+            elif [[ -f /etc/SDE-VERSION ]]; then
+                distro="$(< /etc/SDE-VERSION)"
+                case $distro_shorthand in
+                    on|tiny) distro="${distro% *}" ;;
+                esac
+
             elif type -p crux >/dev/null; then
                 distro=$(crux)
                 case $distro_shorthand in
@@ -1346,6 +1346,8 @@
             has "alps"       && tot alps showinstalled
             has "butch"      && tot butch list
             has "bonsai"     && tot bonsai list
+	    has "mine"       && tot mine -q
+
 
             # Counting files/dirs.
             # Variables need to be unquoted here. Only Bedrock Linux is affected.
